_posts/2016-12-02-one-year-on-gcp.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "One year on Google Cloud Platform"
+ghavatar: 310118
+ghname: gronnbeck
+jobtitle: Platform Engineer
+tags: [cloud, infrastructure, google cloud platform, gcp]
+---
+
+First, let me be frank and announce that we’ve been on the Google Cloud Platform (GCP) for little more than a year. And that I’ve not been working full time with GCP for a year, yet. Nonetheless, I feel ready to share some insights and thoughts on building microservices on GCP.
+
+In this blog post will focus on why we at Unacast chose GCP as our cloud provider, why it’s still a good fit for us, and a few lessons learned of using the platform for about a year.
+
+## Why Google Cloud Platform
+
+Today the natural choice of cloud is Amazon Web Services (AWS). And with good reason. AWS pioneered many of the great cloud services out there, S3, EC2, Lambda etc. It has as far as I know the longest list of cloud components you can use to build your platform <sup>1</sup>. And it’s battle tested at scale by Amazon, Netflix, AirBnB and many others. So why did we choose GCP instead?
+
+Actually, we didn’t. Unacast started out building its platform on a combination of Heroku and AWS. And after some fumbling, sessions of banging our heads against the wall, and some help from a consultant<sup>2</sup> we decided to try GCP. And with some effort and a lot of luck it turned out to be the right platform for us. The reason being two fold 1) GCPs big data capabilities and 2) it helps us minimise time used on operations.  
```

Comment:
For all I know there was a deliberate decision to try GCP. 

We had listed a set of questions to NordCloud, all of which turned out to being ops'y (infrastructure, network, IAM, ...). Based on this NordCloud thought GCP could be a great fit as the hosted services available seemed to match good with our use case.